core/client/fs/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -523,28 +525,27 @@ public void needsSync(AlluxioURI path)
   }
 
   @Override
-  public boolean submitLoad(AlluxioURI path, java.util.OptionalLong bandwidth,
-      boolean usePartialListing, boolean verify) {
+  public Optional<String> submitJob(JobRequest jobRequest) {
```

Comment:
After changing the client API from submitLoad to submitJob, will it have backward compatible problem when users use the old client version?